### DISCUSSION TITLE
Iperf3 in burst mode

### DISCUSSION THREAD

**Discussion by mgb-gvd** [esnet/iperf#1349] [Q&A]
Hi all

I've used the stable windows release (v3.1) to do some UDP burst testing.
This is the command I use:
**iperf3.exe -c 192.168.10.6 -u -l 100 -b /25**
This gave me a burst of approx. 25 datagrams every second during 10s (by default)

Now I've switched to Iperf3.11 and the same command gives me just a burst of 25 datagrams and that's it for the rest of the 10s

I assume there was a conceptual change on the burst-test idea?
But for my specific test I liked the 3.1 way better. Is there a way to get it to work on v3.11 in the same manner?

Thanks

Gerrit

**Comment by davidBar-On** [comment]
Hi Gerrit, this is strange.  I don't see the issue in my machine.  Can you run the client with `-d -V` and send the full client log?
